### PR TITLE
Updated tuning parameter check script

### PR DESCRIPTION
**Pull request by devendra104**
Updated few unavailable checks in the existing check-perf-tuning file.

### CODE REVIEW

File: check-perf-tuning [hunk 1]
```diff
@@ -62,8 +65,9 @@ if [ -f /etc/httpd/conf.modules.d/passenger_extra.conf ]; then
   echo "Configuration : /etc/httpd/conf.modules.d/passenger_extra.conf"
   echo "Custom-hiera  : apache::mod::passenger::passenger_max_pool_size"
   echo "                apache::mod::passenger::passenger_max_request_queue_size"
+  echo "                apache::mod::passenger::passenger_max_request"
```
Comment (sthirugn) [inline check-perf-tuning]: this should be `requests`

**Comment by sthirugn** [comment]
@devendra104 thank you for the PR.  I made one minor comment after which this can be merged.

**Comment by sthirugn** [comment]
@devendra104 do you mind fixing this or close it if not required?

File: check-perf-tuning (hunk 1 above)
Comment (devendra104) [reply check-perf-tuning]: Fixed

**Comment by sthirugn** [comment]
Thank you @devendra104

**Pull request merged**